pallets/vc-management/src/lib.rs

```diff
@@ -165,5 +227,62 @@ pub mod pallet {
 			Self::deposit_event(Event::SomeError { func, error });
 			Ok(Pays::No.into())
 		}
+
+		// Change the schema Admin account
+		#[pallet::weight(195_000_000)]
+		pub fn set_schema_admin(
+			origin: OriginFor<T>,
+			new: T::AccountId,
+		) -> DispatchResultWithPostInfo {
+			T::SetAdminOrigin::ensure_origin(origin)?;
+			Self::deposit_event(Event::SchemaAdminChanged { old_admin: Self::schema_admin() });
+			<SchemaAdmin<T>>::put(new);
+			Ok(Pays::No.into())
+		}
+
+		// It requires the schema Admin account
+		#[pallet::weight(195_000_000)]
+		pub fn add_schema(
+			origin: OriginFor<T>,
+			hash: H256,
```

Comment:
I have two concerns here:
- how is `hash` calculated? Is it by hashing the content? If so we probably either need to check it, or we simply don't pass it as a parameter
- Is `hash` the best option for the storage key? It's a bit counter-intuitive if two schemas have the same id but differ in content

Reply:
The `hash` here is calculated from content. But an auto-increased ID would be better. Will change the hash to ID.